### ISSUE TITLE
Accessibility: Domain request form Type of org radio screen reader output

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2568]
### Problem
The `What kind of U.S.-based government organization do you represent?` radio field has this ANDI / screen reader output:
`Generic org type * Federal: an agency of the U.S. government’s legislative, executive, or judicial branches not checked, required, 1 of 8`

The `Generic org type *` part should reference the form label (`What kind of U.S.-based government organization do you represent?`). This fails section 508 success criteria (SC) 1.3.1-programmatic-label.
![Image](https://github.com/user-attachments/assets/743c96ba-5ea5-4080-a910-d7d5f69b5869)

### Solution
The ANDI output should be:
`What kind of U.S.-based government organization do you represent? * Federal: an agency of the U.S. government’s legislative, executive, or judicial branches not checked, required, 1 of 8`

**Comment by abroddrick** [comment]
@AnnaGingle this and the other accessibility tickets you just made should get the "Dev" label correct? I imagine there isn't any additional design/product input needed prior to devs doing these, but correct me if I am wrong.

**Comment by AnnaGingle** [comment]
This is a dev ticket

**Comment by AnnaGingle** [comment]
This issue will be completed as a part of issue #2848

**Comment by AnnaGingle** [comment]
The fieldset or legend should be: What kind of U.S.-based government organization do you represent? 
The label should be: Federal: an agency of the U.S. government’s legislative, executive, or judicial branches 
required should be: selected, required, 1 of 8
